backend/src/email/middlewares/email-callback.middleware.ts

```diff
@@ -0,0 +1,36 @@
+import { Request, Response, NextFunction } from 'express'
+import { EmailCallbackService } from '@email/services'
+
+const isAuthenticated = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Response | void => {
+  const authHeader = req.get('authorization')
```

Comment:
note to self: we need to enable subscription confirmation to this endpoint first, then do a second release to get rid of the subscription confirmation code.